controllers/storageconsumer/consumer_test.go

```diff
@@ -59,6 +62,10 @@ func createFakeScheme(t *testing.T) *runtime.Scheme {
 	if err != nil {
 		assert.Fail(t, "failed to add configv1 scheme")
 	}
+	err = rookcephv1.AddToScheme(scheme)
+	if err != nil {
+		assert.Fail(t, "failed to add configv1 scheme")
+	}
```

Comment:
Prefer scoped error variables
```suggestion
	if err = rookcephv1.AddToScheme(scheme); err != nil {
		assert.Fail(t, "failed to add configv1 scheme")
	}
```